fluent-react/src/localization.ts

```diff
@@ -73,6 +79,144 @@ export class ReactLocalization {
     return fallback || id;
   }
 
+  getElement(
+    componentToRender: ReactElement,
+    id: string,
+    args?: {
+      vars?: Record<string, FluentVariable>,
+      elems?: Record<string, ReactElement>,
+      attrs?: Record<string, boolean>;
+    },
```

Comment:
```suggestion
    args: {
      vars?: Record<string, FluentVariable>,
      elems?: Record<string, ReactElement>,
      attrs?: Record<string, boolean>;
    } = {},
```

This removes the need to continuously check e.g. `args?.vars`.

Reply:
Good one: dc130a591aaaa4023650b7be16808f38d70ed1be